templates/suggest-image-form.html.jinja2

```diff
@@ -119,8 +119,14 @@ function form() {
     <div class="pm-UploadForm-row">
       <label for="description">Description</label>
       <textarea style="width:100%" id="description" name="description" x-model="description" maxlength="1000"></textarea>
-    </div>
 
+    </div>
+      <p>
+        <em>**A puzzle instance is a memory slot that you have available to store a puzzle. You will have
+          full administrative control over this puzzle. These can be gained by donating to the website.
+          Please click the link below to purchase a puzzle instance!</em>
+      </p>
+    <a href="http://puzzle.massive.xyz/d/buy-stuff/">Buy Instance Slot</a>
```

Comment:
This page has actually changed to be more contribution focused.  I can reword the purchase part and change it later.